.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- Thank you for working on Jamulus and opening a Pull Request! Please fill the following to make maintenance and development process visible -->
+
+**Short description of changes**
+
+<!-- A short description of your changes which might go into the change log -->
+
+**Context: Fixes an issue?**
+
+<!-- If this fixes an issue, please write Fixes: <issue number here>; if not, please give your PR a context. -->
+
+**Does this change need documentation? What needs to be documented and how?**
+
+<!-- Most new features should be documented on the website: https://github.com/jamulussoftware/jamuluswebsite/ If you have a proposal what to document, feel free to open a draft PR on the website repo -->
+
+**Status of this pull request**
+<!-- This might be edited by maintainers. -->
+<!-- Proof of concept (not to be merged soon); Working implementation; ... -->
+
+**What is missing until this pull request can be merged?**
+<!-- Does it still need more testing; ... -->
+
+## Checklist
+- [ ] I verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
+- [ ] I tested my code and it does what I want
```

Comment:
Sounds a bit vague. Should we make it more specific? Some ideas:
- I have documented the platforms I've tested this PR on.
- I have documented how this PR can be tested and its correctness can be verified.
?

Reply:
Agreed.

Jamulus does actually have a test suite...  Ideally, we'd be maintaining it.

For all parts of the system not covered (most of the program), we'd still write out the spec / test guideance notes used, as part of the discussion or PR itself.

But still, it's not going to stop code getting reviewed and accepted.